.drone.yml

```diff
@@ -258,6 +258,8 @@ pipeline:
     image: plugins/gcr
     repo: eminent-nation-87317/vic-machine-server
     dockerfile: cmd/vic-machine-server/Dockerfile
+    environment:
+      - DRONE_TAG:  ${DRONE_TAG}
```

Comment:
Looks like the correct format for specifying env variables doesn't use the `-` character (see the `vic-integration-test-on-pr` step).

Reply:
ah thanks for pointing that out, fixed!